python/requirements/ml/requirements_tune.txt

```diff
@@ -40,3 +40,8 @@ transformers==4.10.0
 wandb==0.12.5
 xgboost==1.3.3
 zoopt==0.4.1
+# Dependencies for Hugging Face examples:
```

Comment:
@richardliaw ptal

Reply:
This was a result of a faulty merge and shouldn't be added. See https://github.com/ray-project/ray/pull/20767